### DISCUSSION TITLE
HomeAssistant MQTT / MG5050 / IP150 / PAI 3.3.0

### DISCUSSION THREAD

**Discussion by DGNoyon** [ParadoxAlarmInterface/pai#412] [Q&A]
Hi everyone,
On my MG5050, we can arm with: arm, arm_away, arm_night and arm_home but I would like to be able to arm_vacation because I would like to add some automation when we are absent for a long period (light on, TV etc...)
I tried to add in the configuration in his MQTT_COMMANDS_ALIAS the line: "armed_vacation: arm" but it doesn't work.
So, it is possible to emulate this arm_vacation ?

**Comment by yozik04** [comment]
I do not think that the panel supports so many different arming types.

**Comment by DGNoyon** [comment]
Yes, true, in is not provided by the panel but through HA, I would like to implement it to arm the alarm like done by arm_away + doing some HA automations

**Reply by yozik04** [reply]
Use Scenes? Maybe.

**Reply by DGNoyon** [reply]
I will try ... thanks
